tasks/request_files/request_files.py

```diff
@@ -365,6 +347,40 @@ def inner_task(
         "asyncOperationId": event[EVENT_CONFIG_KEY][CONFIG_JOB_ID_KEY],
     }
 
+def get_glacier_recovery_type(config: Dict[str, Any]) -> str:
+    """
+    Returns the glacier recovery type from either config or environment variable. 
+    Must be either 'Bulk', 'Expedited', or 'Standard'.
+    Args:
+        config: The config dictionary from lambda event.
+
+    Raises: ValueError if recovery type value is invalid.
+    """
+
+
+    VALID_RESTORE_TYPES = ["Bulk", "Expedited", "Standard"]
+
+    # Look for config override
+    retrieval_type = config.get(CONFIG_RESTORE_DEFAULT_RECOVERY_TYPE_OVERRIDE_KEY, None)
+    if retrieval_type is not None:
+        if retrieval_type in VALID_RESTORE_TYPES:
+            LOGGER.info(f"Found glacier restore type from config: {retrieval_type}. Overridding other values.")
+        else:
+            LOGGER.error(f"Invalid value in restore type {retrieval_type} in config.")
+            raise ValueError("Invalid value in config. Valid values are 'Bulk', 'Standard', 'Expedited'")
+
+    # Look for default from TF
+    if retrieval_type is None:
+        retrieval_type = os.getenv(OS_ENVIRON_RESTORE_DEFAULT_RECOVERY_TYPE_KEY, None)
+        if retrieval_type is not None:
+                if retrieval_type in VALID_RESTORE_TYPES:
+                    LOGGER.info(f"Found restore type from {OS_ENVIRON_RESTORE_DEFAULT_RECOVERY_TYPE_KEY}: {retrieval_type}")
+                else:
+                    LOGGER.error(f"Invalid value in restore type {retrieval_type} in env variable.")
```

Comment:
Possibly change `"Invalid value in restore type {retrieval_type} in env variable."` to `f"Invalid restore type value of {retrieval_type} found in environment variable {OS_ENVIRON_RESTORE_DEFAULT_RECOVERY_TYPE_KEY}."`

Reply:
updated